_pages/clad.md

```diff
@@ -73,6 +73,186 @@ f_dy_dx.execute(x, y, result2);
 std::cout << "dy: " << result2[0] << ' ' << "dx: " << result2[1] << std::endl;
 ```
 Note: *we are working on improving the gradient interface*.
+
+### Vector forward mode
```

Comment:
I meant to ask to move it in the clad repo and to show up at [clad docs.](https://clad.readthedocs.io/)

Reply:
ah, my bad. Moved it here: https://github.com/vgvassilev/clad/pull/809
This PR now just removes the page and images.